clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java

```diff
@@ -63,7 +63,7 @@ public long writeTo(GatheringByteChannel channel, long previouslyWritten, int re
                             " converted_records_size: " + sizeOfFirstConvertedBatch);
 
                 recordConversionStats.add(recordsAndStats.recordConversionStats());
-                log.debug("Got lazy converted records for {" + topicPartition() + "} with length=" + convertedRecords.sizeInBytes());
+                log.debug("Got lazy converted records for topic-partition {} with length={}", topicPartition(), convertedRecords.sizeInBytes());
```

Comment:
Nit: we typically just say `partition` in these cases. Same for the other log.debug.